lib/server.js

```diff
@@ -1570,7 +1586,20 @@ WalletService.prototype.getPendingTxs = function(opts, cb) {
       txp.deleteLockTime = self.getRemainingDeleteLockTime(txp);
     });
 
-    return cb(null, txps);
+    async.each(txps, function(txp, a_cb) {
+      if (txp.status != 'accepted') return a_cb();
+
+      self._checkTxInBlockchain(txp, function(err, tx) {
```

Comment:
Since the `txid` is stored in the proposal, I think this method should return `true`/`false`. It should also use `txp.txid` instead of rebuilding the bitcore transaction.